src/Rules/PhpDoc/AssertRuleHelper.php

```diff
@@ -55,6 +56,22 @@ public function check(ExtendedMethodReflection|FunctionReflection $reflection, P
 				continue;
 			}
 
+			if ($assert->getType() instanceof ErrorType) {
```

Comment:
Call UnresolvableTypeHelper instead please. You should follow the code samples I sent you :)

Reply:
Also please add all the other things that should be checked too.